Dockerfile

```diff
@@ -87,15 +97,21 @@ ARG RUNDIR
 ARG SKIP_TESTS
 ARG SKIP_PRUNE
 
+
+LABEL br.lnls.epics-in-docker.skip-prune=${SKIP_PRUNE}
+LABEL br.lnls.epics-in-docker.app-dirs=${APP_DIRS}
+LABEL br.lnls.epics-in-docker.rundir=${RUNDIR}
```

Comment:
I've tested with ad-sim-epics-ioc, and this approach actually doesn't work. :/

When we COPY part of the filesystem, these labels won't be inherited. We must define them in the `*-link` stages.